client/src/components/pages/member/index.js

```diff
@@ -22,7 +22,7 @@ class MemberPage extends Component {
       let member, socialMedia;
 
       if (!this.props.member) {
-          return <div/>
```

Comment:
We should eventually replace the empty div/no one here h1 with a loading component as it fetches data from the server